engine/src/renderer/vulkan/vulkan_backend.c

```diff
@@ -165,8 +171,11 @@ b8 vulkan_renderer_backend_initialize(renderer_backend* backend, const char* app
     VK_CHECK(vkCreateInstance(&create_info, context.allocator, &context.instance));
     KINFO("Vulkan Instance created.");
 
-    // Debugger
 #if defined(_DEBUG)
+    darray_destroy(required_validation_layer_names);
+    darray_destroy(required_extensions);
```

Comment:
Shouldn't these two destroy calls be outside the _DEBUG block?

Reply:
Ah, good catch -- the required_extensions destroy call needs to be moved out of that block.  The required_validation_layer_names one should remain there as it is created within another _DEBUG block.